console/playwright/playwright.config.ts

```diff
@@ -1,12 +1,18 @@
 import type { PlaywrightTestConfig } from '@playwright/test'
 import { devices } from '@playwright/test'
+import * as fse from 'fs-extra'
 
 /**
  * Read environment variables from file.
  * https://github.com/motdotla/dotenv
  */
 require('dotenv').config()
 
+fse.emptyDir('test-video')
+fse.emptyDir('test-storage')
+fse.ensureDir('test-video')
+fse.ensureDir('test-storage')
```

Comment:
remove `ensureDir` calls?

https://github.com/jprichardson/node-fs-extra/blob/master/docs/emptyDir.md

> emptyDir(dir[, callback])
Ensures that a directory is empty. Deletes directory contents if the directory is not empty. If the directory does not exist, it is created. The directory itself is not deleted